index.js

```diff
@@ -42,6 +42,10 @@ Toolkit.run(
           tools.log('PR title ' + title + ' does not contain approved project with format [PROJECT-1234]')
           return false
         }
+        if (title.match(createMappedBranchRegex(head_branch))) {
+          tools.log('PR title ' + title + ' contain unexpected branch parts.')
```

Comment:
maybe its not clear to the user what this "contain unexpected branch parts." I would probably print something like "PR title is ambiguous, please set a more descriptive title"